README.md

```diff
@@ -1,88 +1,127 @@
+    Open your browser to http://localhost:8080/
+    You should now be able to see and interact with the project
 
-###### [Frontend Interface](https://github.com/josh-everett01/travel_tracker)
+# Tech Stack
 
-### Versions
+- JavaScript
+- Sass / CSS
+- HTML
+- Webpack - Version 4.31
 
-`npm 6.14.7`
+# Learning Objectives
 
-### Tech Stack
+####The purpose of my building this project was to build an application that would:
 
-- JavaScript
-- HTML5/CSS3/SCSS
+##### - Facilitate the movement of data to-and-from an API endpoint
+
+- This was done by using JavaScript to authenticate user login, to fetch trip/destination data, and to post / delete trips via their appropriate endpoints.
+
+##### - Display the data to a user
+
+- To do this I created a User Interface that allowed the data to be rendered visibily for the user to view and understand. This was done using HTML / CSS / and JavaScript.
+
+##### - Allow the user to interact with the data
+
+- Once a user logs in via a login form they are then able to interact with the data by either making a trip request as a traveler or having the ability to approve pending trips, search for a traveler, and approve / deny trips for a traveler as an agent.
+
+##### - Be fully responsive on all devices
+
+- This was done by using media queries to make sure the page is rendered correctly when viewed on different devices.
+
+##### - Facilitate my better understanding of some of the concepts associated with having a job in the web development field
+
+- This was done by using a project board to keep track of the tasks associated with the project and to track the progress of the project. Every task had an issue created and that issue would then go through the process of being checked out, worked on, reviewed, and then approved to be merged into the project.
+
+- I would reach out to either an online group or a mentor for guidance when I reached a point where I could not progress any further due to not being able to find the answer on my own.
+
+- Through this project I have learned how to use git to my advantage in many ways. Most notably and what took the most practice was my learning to squash my commits into one clean-and-readable commit and then rebase to master. This helps the reviewer when reviewing pull requests.
+
+# Credits
+
+####I would not have been able to finish this project without the assistance of:
+
+[Rob Stringer](https://github.com/Mycobee) reviewing pull requests, and providing support and guidance on all aspects of this project.
+[Colin McDonnell](https://github.com/colinmcdonnell) providing support and guidance with multiple JavaScript and CSS issues during this project.
+The group @ [JavaScript Tutoring](https://javascripttutoring.slack.com) providing support and offered assistance during this project.
+[Mozilla JavaScript Documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript)
+[W3 Schools JavaScript Documentation](https://www.w3schools.com/js/default.asp)
```

Comment:
Turing school open source curriculum and the contributors to the starter webpack scaffold/project objectives